### PR TITLE
spec file: add python-netaddr dependency for Opensuse 42.2

### PR DESCRIPTION
**Pull request by jan--f**
Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

**Comment by jan--f** [comment]
Fixes: #148

File: deepsea.spec [hunk 1]
```diff
@@ -32,6 +32,9 @@ BuildRequires:  salt-master
 Requires:       salt-master
 Requires:       salt-minion
 Requires:       python-ipaddress
+%if 0%{?leap_version} == 420200
```
Comment (rjfd) [inline deepsea.spec]: Should we also check if this is necessary in SLE?

Comment (jan--f) [inline deepsea.spec]: sure but lets do it in another PR. This is blocking me in Teuthology.

Comment (jan--f) [inline deepsea.spec]: @rjfd btw I checked and in sle the salt-master brings in python-netaddr as a dependency
so we should keep an eye on that when sle updates the salt-master.
